client/components/recycleAndReuseComponents/AddItemMultiSelect.js

```diff
@@ -30,13 +63,21 @@ const AddItemMultiSelect = ({ data, setItems, openDialog }) => {
 		event.preventDefault();
 		const selectedItems = getSelectedItems(event.target.elements);
 		setItems(selectedItems);
-		openDialog();
+		willTriggerDialog(true);
 	};
 
 	return (
 		<form onSubmit={handleSubmit} style={{ width: "100%" }}>
+			{!!selectedOptions && selectedOptions.length ? (
```

Comment:
this block is the main added logic right? this block can stay

Reply:
hey sorry for the delay , yup ok not sure why theres alot of other changes also , will close this PR and redo this portion